module3/projects/consultancy/inception.md

```diff
@@ -0,0 +1,184 @@
+---
+layout: page
+title: Consultancy - Inception
+type: project
+---
+_[Back to Consultancy Home](./index)_ 
+## Background
+
+An Inception is a process used by many companies in the Software Industry to design a solution to a problem. It is tempting to take a problem and immediately start coding, however this will often result in an incoherent solution, or even worse, a solution to the **wrong** problem.
+
+Usually this process takes several days involving many stakeholders such as the client(s), team leads, CEOs, operations, etc. We will be doing a simplified version of this process in a few hours between your team and your project manager.
+
+## Inception Deliverables
+
+These items must be shared with your project manager before you can begin development. They will be reviewed in your first project check-in as well. Please pin or bookmark these in your Slack channel. 
+
+1. Slack Channel Created
+2. DTR
+3. Miro Board or Planning Tool that includes:
+    - MVP definition
+    - Wireframes
+    - Chosen external API
+    - Chosen BE stretch tech
+    <!-- - React Component Architecture DO WE WANT THIS? -->
+    - Database Schema
+4. User Stories in Project Board
+5. Equity Analysis
+6. Plan for sub-teams for first development sprint
+    - Team rotation is required after sprints
```

Comment:
Agreed - I think it'll be even more important to have students work on both BE and FE in this project because they will be different frameworks... but I've definitely seen teams struggle to do rotations even when it's all in Rails. One alternative is to require students to write "vertical" user stories that are full features that include any BE updates, but I worry that this could be too difficult for them to manage without prior experience with full-stack user stories.

Reply:
I like the idea of vertical stories. They're going to be doing both ends of the stack for the final project/assessment, so it's not a bad idea to encourage a first attempt at developing a whole feature prior to the final.

Reply:
Cool - let's try vertical splitting stories! I'll change this line to `6. Feature and user story assignment for first sprint`